src/config/index.ts

```diff
@@ -1,6 +1,11 @@
+import dotenv from 'dotenv';
+
 export { prettifyConfig } from './sqlPrettifyConfig';
 import { version } from '../../package.json';
 
+if (getConfig('NODE_ENV') !== 'production') {
+  dotenv.config({ path: '.env.dev' });
+}
```

Comment:
I moved this from the root/index.ts because it was constantly getting an eslint error: "There should be no empty line within import group"